### ISSUE TITLE
a11y: Improve modal popup accessibility

### ISSUE THREAD

**Issue by tshfx** [MetaMask/metamask-extension#6863]
**Describe the bug**
When a user is navigating the extension by tabbing (or a screen reader) the extension tends to make a user tab through everything hidden behind before being able to tab through the content presented.

The user is also unable to close the modal.

**To Reproduce**
1. Tab select Details.
2. The details modal pops up
3. Try to select "View on Etherscan"
4. The user must tab many times.

**Expected behavior**
Tabbing should prioritize a modal if present. A user should be able to close the modal.

**Screenshots**
![Screen Shot 2019-07-15 at 9 52 26 PM](https://user-images.githubusercontent.com/52763493/61258218-df764780-a74a-11e9-87bc-ff13dcee1c98.png)

**Browser details (please complete the following information):**
 - OS: OS X
 - Browser: chrome
 - MetaMask Version: 6.7.2

**Additional context**
A11y refers to accessibility. This change would improve the extensions screen reader support.

**Comment by Gudahtt** [comment]
The address is now auto-focused when the modal opens, and the close button is now focusable.

There are still improvements to be made (e.g. you can still focus things in the background), but the basic issue here has been resolved.
